### PR TITLE
4660 custom analytics config

### PR DESCRIPTION
**Pull request by mheppler**
## Related Issues

- connects to #4660 Custom Analytics - More than just pageviews

## Pull Request Checklist

- [ ] Unit [tests][] completed
- [ ] Integration [tests][]: None
- [x] Deployment requirements, [SQL updates][], [Solr updates][], etc.: None
- [x] [Documentation][docs] completed
- [x] Merged latest from "develop" [branch][] and resolved conflicts

[tests]: http://guides.dataverse.org/en/latest/developers/testing.html
[SQL updates]: http://guides.dataverse.org/en/latest/developers/sql-upgrade-scripts.html
[Solr updates]: https://github.com/IQSS/dataverse/blob/develop/conf/solr/7.3.0/schema.xml
[docs]: http://guides.dataverse.org/en/latest/developers/documentation.html
[branch]: http://guides.dataverse.org/en/latest/developers/branching-strategy.html

### CODE REVIEW

File: src/main/webapp/dataset.xhtml [hunk 1]
```diff
@@ -63,7 +63,7 @@
                                                                               and !DatasetPage.dataset.deaccessioned
                                                                               and DatasetPage.showComputeButton()}">
                                 <!-- Compute Button Group -->
-                                <button type="button" id="computeBatch" class="btn btn-default dropdown-toggle" data-toggle="dropdown">
+                                <button type="button" id="computeBatch" class="btn btn-default btn-compute dropdown-toggle" data-toggle="dropdown">
```
Comment (pdurbin) [inline src/main/webapp/dataset.xhtml]: This "compute" change feels a bit unrelated but whatever. Cleanup, maybe?

File: src/main/webapp/dataset.xhtml [hunk 2]
```diff
@@ -598,7 +598,7 @@
                                         <span class="glyphicon glyphicon-pencil"/> #{bundle['file.dataFilesTab.metadata.addBtn']}
                                     </p:commandLink>                                   
                                     <div class="btn-group" jsf:rendered="#{DatasetPage.dataset.released and !DatasetPage.dataset.deaccessioned}" >
-                                        <button class="btn btn-default dropdown-toggle" type="button" styleClass="btn btn-default" data-toggle="dropdown">
+                                        <button class="btn btn-default btn-export dropdown-toggle" type="button" styleClass="btn btn-default" data-toggle="dropdown">
```
Comment (pdurbin) [inline src/main/webapp/dataset.xhtml]: Actually, I guess all of these are cleanup?

File: src/main/webapp/dataverse.xhtml [hunk 3]
```diff
@@ -473,7 +473,7 @@
                                     <f:setPropertyActionListener target="#{sendFeedbackDialog.recipient}" value="#{DataversePage.dataverse}"/>
                                         <span class="glyphicon glyphicon-envelope"/> #{bundle['contact.contact']}
                                 </p:commandLink>
-                                <p:commandLink type="button" styleClass="text-button bootstrap-button-tooltip"
+                                <p:commandLink type="button" styleClass="text-button btn-share bootstrap-button-tooltip"
```
Comment (pdurbin) [inline src/main/webapp/dataverse.xhtml]: More cleanup?

**Comment by mheppler** [comment]
In reply to code review comments from @pdurbin, those are not "clean up" changes to the buttons, but the addition of new style classes to end user event buttons (e.g. Contact, Share, Compute, Download...) to aid in the tracking of those events with analytics. I had added a comment to the issue regarding the inclusion of them in order to deliver on a user request which was the origin of this custom analytics issue. They were first discussed with @kcondon, who suggested I review it with @djbrooke, who then approved including it in this issue to deliver more value to our sysadmins.

**Comment by pdurbin** [comment]
@mheppler my mistake. Thanks for the clarification! I see (now) the comment you're talking about: https://github.com/IQSS/dataverse/issues/4660#issuecomment-449058697

**Pull request merged**